core/common/src/main/java/alluxio/PropertyKey.java

```diff
@@ -895,6 +895,16 @@ public String toString() {
               + "href=\"#configure-multihomed-networks\">multi-homed networks</a>.")
           .setScope(Scope.MASTER)
           .build();
+  public static final PropertyKey MASTER_CONFIG_REPORT_MAX_LOG_INTERVAL_MS =
+      new Builder(Name.MASTER_CONFIG_REPORT_MAX_LOG_INTERVAL_MS)
+          .setDefaultValue("1h")
+          .setDescription("If the time gap between the current time and "
```

Comment:
We are now using this property for two different purposes: (1) as a min interval between logging non-status-changing config report updates and (2) as the heartbeat interval for regularly logging the config report.

I don't think users will care to configure (1), it is really only useful for debugging or testing purposes. We will probably eventually solve the problem of (1) by only logging incremental changes to the report. For now, can we make it a hidden property?

Users might care about (2) since 1 hour could be too often for some use cases (for example, they have lots of warn-level properties that they actually want to be inconsistent).

Can we go back to using two properties so that we can address the two different uses of this property separately?